tests/vault/test_vault_symmetric.yml

```diff
@@ -178,6 +178,33 @@
     register: result
     failed_when: result.data != 'Hello World.' or result.changed
 
+  - name: Change vault password.
+    ipavault:
+      ipaadmin_password: SomeADMINpassword
+      name: symvault
+      password: SomeVAULTpassword
+      new_password: SomeNEWpassword
+    register: result
+    failed_when: not result.changed
```

Comment:
I'd also add a test to retrieve data using the new password

Reply:
Added a test for that.